kuma/core/tests/test_views.py

```diff
@@ -1,40 +1,47 @@
-from nose.tools import eq_
+import logging
+from nose.tools import eq_, ok_
 from pyquery import PyQuery as pq
 from soapbox.models import Message
 
-from django.conf import settings
 from django.core import mail
+from django.test import override_settings
+from django.utils.log import AdminEmailHandler
 
 from kuma.core.tests import KumaTestCase
 
 from ..urlresolvers import reverse
 
 
+@override_settings(
```

Comment:
Nice use of ``@override_settings``, and happy to see logging garbage out of test output